apps/application-system/form/src/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
Why does this keep popping up ?

Reply:
👀 https://github.com/island-is/island.is/blob/main/.gitignore#L83

Reply:
Not sure it happens every time I run yarn format :/